packages/site/src/config/snap.ts

```diff
@@ -2,4 +2,5 @@
  * The snap origin to use.
  * Will default to the local hosted snap if no value is provided in environment.
  */
-export const defaultSnapOrigin = 'npm:@metamask/snap-simple-keyring-snap';
+export const defaultSnapOrigin =
```

Comment:
should we also have if __DEV__ check to set this value? will the `packages/site/.env.production` always be present when developing locally?

Reply:
The way it's configured, `packages/site/.env.production` will always be there.  Another way to approach this would be to also make a `.env.development`, but I'm just going by what the authors of  template-snap-monorepo set up.

The important case here is probably not actually local `yarn build`, but CI `yarn build`, which we won't see for sure until the next release is built.